samples/create_endpoint_sample.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_create_endpoint_sample]
+from google.cloud import aiplatform
+
+
+def create_endpoint_sample(display_name: str, project: str):
+    client_options = dict(api_endpoint="us-central1-aiplatform.googleapis.com")
+    client = aiplatform.EndpointServiceClient(client_options=client_options)
+    location = "us-central1"
+    parent = "projects/{project}/locations/{location}".format(
+        project=project, location=location
+    )
+    endpoint = {"display_name": display_name}
+    response = client.create_endpoint(parent=parent, endpoint=endpoint)
+    print("Long running operation:", response.operation.name)
+    create_endpoint_response = response.result(timeout=300)
```

Comment:
If the default timeout is too low I would also edit the gRPC service config.

Reply:
I still stand by this comment - does the gRPC service config need to be changed?

Reply:
This is a little tricky.  In a new project (where an endpoint has not been created), the first time this method is called it could take up to 300 seconds, but subsequent calls in the same project finishes within seconds.  Since GAPIC library internally polls the long running operation, the tests typically do not run that long.

Agreed that perhaps it's better to change the timeout at the gRPC service config.